riotfile.py

```diff
@@ -722,10 +722,8 @@ def select_pys(min_version=MIN_PYTHON_VERSION, max_version=MAX_PYTHON_VERSION):
                     },
                 ),
                 Venv(
-                    # celery added support for Python 3.10 in 5.2, no official support for 3.11 yet
-                    # Billiard dependency is incompatible with Python 3.11
-                    # https://github.com/celery/billiard/issues/377
-                    pys="3.10",
+                    # celery added support for Python 3.10 in 5.2
+                    pys=select_pys(min_version="3.10"),
```

Comment:
do we need to specify the celery version we want to use to run with 3.11? I don't think that celery actually supports 3.11 yet does it?

Reply:
https://pypi.org/project/celery/ it looks like it only goes up to 3.10?

Reply:
ah nvm, read the gh issue. I think we still need to specify running celery 5.3 with python 3.11 though right?